src/module.c

```diff
@@ -531,6 +531,69 @@ Module *Module::parse()
             error("has non-identifier characters in filename, use module declaration instead");
     }
 
+    // Add internal used functions in 'object' module members.
+    if (!parent && ident == Id::object)
+    {
+        static const utf8_t code_ArrayEq[] =
+            "bool _ArrayEq(T1, T2)(T1[] a, T2[] b) {\n"
+            " if (a.length != b.length) return false;\n"
+            " foreach (size_t i; 0 .. a.length) { if (a[i] != b[i]) return false; }\n"
+            " return true; }\n";
+
+        static const utf8_t code_ArrayPostblit[] =
+            "void _ArrayPostblit(T)(T[] a) { foreach (ref T e; a) e.__xpostblit(); }\n";
+
+        static const utf8_t code_ArrayDtor[] =
+            "void _ArrayDtor(T)(T[] a) { foreach_reverse (ref T e; a) e.__xdtor(); }\n";
```

Comment:
Do you mean `_destructRecurse` and `_postblitRecurse`? I also planned them, but i decided not to use them.
Because they're specialized to `T[n]`. It will instantiate different templates for T[1], T[2], ..., so they would cause template bloat.